console_run.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import os
+import signal
+import warnings
+from datetime import datetime, timedelta
+
+from LatiaoAid import LatiaoAid
+from Logger import Logger
+
+
+def parse_args() -> argparse.Namespace:
+    """Parse the command line arguments for the `LatiaoAid` binary.
+
+    :return: Namespace with parsed arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Collects 辣条/亲密度 on live.bilibili.com using selenium webdriver"
+    )
+
+    parser.add_argument(
+        "-r", "--room", type=int, help="default room", default=22198526,
+    )
+
+    # runtime behaviors
+    parser.add_argument(
+        "--headless", help="do not show the browser", action="store_true",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--disable-image",
+        help="do not show the video content",
```

Comment:
With this option, only <img> tags are disabled, videos are not, so it should be "img".